.github/workflows/deployments.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       - name: Determine build info
         id: build_info
         run: |
-          requested_platforms="${{ inputs.platforms || (github.event_name == 'push' && 'linux/amd64,linux/arm64') || 'linux/arm64 linux/amd64' }}"
+          requested_platforms=`echo ${{ inputs.platforms || 'linux/arm64 linux/amd64' }} | tr + ,`
```

Comment:
Thanks for the updated comments. They help. But I'm still a bit confused as to why the default value on line 22 (comma) is different than the fallback value after the `||` specified here on line 99 (space).

More details:
I can't really construct a scenario where the `|| fallback` values is ever used. That is - if the calling script doesn't specify the `platform`, then the default from line 22 will be used and the fallback value is unnecessary. Is it possible that the fallback value was added to this line before the default value was provided in line 22? If so, maybe the fallback value is no longer needed. If, however, it _is_ needed, why does it need to be different than the default value?

Reply:
Ah, the fallback value is used when the trigger of the workflow is not a workflow_dispatch. In this case, inputs.platforms is not defined. Specifically, when a PR is merged or a release branch is created, then the triggering event is not a workflow_dispatch. In that case, this deployment builds the dev dependencies for each paltform separately to update the build caches, and then triggers itself as a workflow_dispatch again in line 212 to build everything as a multiplatform image (not creating any build caches).

Reply:
Ah, thanks for the explanation. That clears it up.